code/drasil-example/glassbr/lib/Drasil/GlassBR/Requirements.hs

```diff
@@ -54,8 +54,11 @@ sysSetValsFollowingAssumpsDesc = foldlSent [atStartNP (the system), S "shall set
     plural value, S "as described in the table for", namedRef sysSetValsFollowingAssumpsTable (S "Required Assignments")]
 
 sysSetValsFollowingAssumpsTable :: LabelledContent
-sysSetValsFollowingAssumpsTable = mkValsSourceTable (mkQRTupleRef r2AQs r2ARs ++ mkQRTuple r2DDs) "ReqAssignments"
-                                  (S "Required Assignments" `follows` sysSetValsFollowingAssumps)
+sysSetValsFollowingAssumpsTable = 
+  mkValsSourceTable 
+    (mkQRTupleRef r2AQs r2ARs ++ mkQRTuple r2DDs)
+    "ReqAssignments"
+    (S "Required Assignments")
```

Comment:
Removes mutual references between the System Sets Immediate Values FR and its table. Opting to keep the one in the FR, removing the backreference in the table.